test/unit/airtap.js

```diff
@@ -0,0 +1,26 @@
+const test = require('tape')
+const path = require('path')
+const exec = require('child_process').exec
+const airtap = path.resolve(__dirname, '../../bin/airtap.js')
+
+test('exits cleanly and does nothing if no secure travis env', t => {
+  const env = Object.assign({}, process.env, {
+    TRAVIS_SECURE_ENV_VARS: 'false'
+  })
+  exec(airtap, { env }, (err, stdout, stderr) => {
+    t.error(err, 'no error')
+    t.equal(stdout.trim(), 'Skipping airtap due to no secure travis environment!')
```

Comment:
I guess it's okay for now, but I don't like how the test depends on a hard-coded string. Do you agree as well? 😅